package.json

```diff
@@ -40,7 +40,8 @@
     "update-versions": "yarn workspace botbuilder-repo-utils update-versions"
   },
   "devDependencies": {
-    "@azure/ms-rest-js": "1.9.0",
+    "@azure/logger": "^1.0.2",
```

Comment:
Where is this used? This should likely go in a `package.json` file within the package (i.e. `libraries/X`) it's used in.

Reply:
Regarding this issue, here's the error we get when building without this dependency
![image](https://user-images.githubusercontent.com/18757147/132907323-ddcfeb75-b426-4943-9f9c-b3e3b6bc2d6e.png)

Identity doesn't have a newer version that makes use of logger 1.0.2 and version 1.0.0 is the problem.

We also tried to use nohoist, but botFramework-connector is not private, so that config was ignored.

Do you have an alternative we can use for this?

Reply:
I also forgot to mention that adding logger 1.0.2 to connector's package.json had no effect

Reply:
I saw your issue, some problem with `browserify`. Easier to take this change than figure out why the bundle process fails.